spec/bolt/cli_spec.rb

```diff
@@ -269,7 +269,7 @@ def stub_config(file_content = {})
         cli.execute(cli.parse)
       end
 
-      it 'install modules from Puppetfile with resolving' do
+      it 'install modules from Puppetfile without resolving' do
```

Comment:
I think a test in `spec/integration` might help keep this file a bit more manageable, especially since we're just checking file contents and errors. But basically the same idea!

Reply:
Moved the new tests to `integration/module_installer_spec`